spec/unit/runner_test.js

```diff
@@ -65,6 +65,6 @@ describe('the Protractor runner', function() {
     var runner = new Runner(config);
     runner.run().then(function() {
       done.fail('expected error when no custom framework is defined');
-    }, done);
+    }, done());
```

Comment:
The "then" function receives a Function as parameter, passing done() to it will call "done" first and make the test always pass, won't it?

Reply:
This test is expecting exception thrown, so if error occurred, done(), test passed.
One big change in Jasmine 3 is done() accept one parameter now, if we pass done function here, done(e) will be called, Jasmine will fail the test. Alternative change here is

```
    runner.run().then(function() {
      done.fail('expected error when no custom framework is defined');
    }, () => {return done();} );
```